<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pcl_visualizer</class>
 <widget class="QMainWindow" name="pcl_visualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pcl_visualizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelimage">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>36</y>
      <width>441</width>
      <height>371</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>461</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>269</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_dir">
       <property name="text">
        <string>open dir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pic">
       <property name="text">
        <string>open_pic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pcl">
       <property name="text">
        <string>open_pcd</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>452</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="previous">
       <property name="text">
        <string>previous</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>268</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="next">
       <property name="text">
        <string>next</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
